extensions/confluence-search/src/index.tsx

```diff
@@ -26,11 +26,10 @@ const init = {
 
 export default function Command() {
   const [results, setResults] = useState<SearchResult[]>([]);
-
+  const [searchText, setSearchText] = useState("");
   const [loadingState, setLoadingState] = useState(true);
-
   useEffect(() => {
-    searchConfluence().then((response) => {
+    searchConfluence(searchText).then((response) => {
```

Comment:
I'm pretty sure this will lead to race conditions where the previous query will maybe resolve after the next one and so you will get outdated results.

You should probably use an `AbortController` to cancel the existing query before doing a new one (see https://github.com/raycast/extensions/blob/main/examples/api-examples/src/typeahead-search.tsx#L43-L90 for an example)

Reply:
Thanks for the shout @mathieudutour ! 
I'll check it out